### PR TITLE
More Analytic Schema Tests

### PR DESCRIPTION
**Pull request by antares1470**
### Prerequisites

- [x] Reviewed the [checklist](CHECKLIST.md)

- [ ] Reviewed feedback from the "Sonar Cloud" bot. Note that you have to wait
    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change

Added unit tests for the remaining classes in Core Analytic Schema requiring testing. Also updated some tests from the previous PR.

Also like the last one, included a small amount of code cleanup.

### Why Should This Be In Core?

More unit tests is better.

### Benefits

More unit tests

### Applicable Issues
#413

### CODE REVIEW

File: CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/attribute/objects/ContractedCompositeNodeState.java [hunk 1]
```diff
@@ -145,7 +145,7 @@ public List<Integer> expand(final GraphWriteMethods wg, final int vxId) {
             wg.setObjectValue(compositeStateAttr, id, new CompositeNodeState(id, expansionState));
         });
 
-        // Correct the x,y,z coordinates of the nodes in the zonsitutentNodeStore in case the composite node has been moved.
+        // Correct the x,y,z coordinates of the nodes in the consitutentNodeStore in case the composite node has been moved.
```
Comment (OrionsGuardian) [inline CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/attribute/objects/ContractedCompositeNodeState.java:148]: Typo: constituentNodeStore

File: CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/compatibility/AnalyticSchemaV5UpdateProvider.java [hunk 2]
```diff
@@ -134,28 +134,28 @@ protected void schemaUpdate(final StoreGraph graph) {
             // K Truss
             if (oldKTrussColorAttributeIdTransaction != Graph.NOT_FOUND) {
                 final int newKTrussColorAttributeIdTransaction = ClusteringConcept.TransactionAttribute.K_TRUSS_COLOR.ensure(graph);
-                final SchemaAttribute oldKTruss = graph.getObjectValue(oldKTrussColorAttributeIdTransaction, transactionId);
```
Comment (OrionsGuardian) [inline CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/compatibility/AnalyticSchemaV5UpdateProvider.java:137]: SchemaAttribute is now an unused import.

File: CoreAnalyticSchema/test/unit/src/au/gov/asd/tac/constellation/graph/schema/analytic/attribute/TransactionTypeAttributeDescriptionNGTest.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.graph.schema.analytic.attribute;
+
+import au.gov.asd.tac.constellation.graph.schema.analytic.concept.AnalyticConcept;
+import au.gov.asd.tac.constellation.graph.value.readables.IntReadable;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.fail;
```
Comment (OrionsGuardian) [inline CoreAnalyticSchema/test/unit/src/au/gov/asd/tac/constellation/graph/schema/analytic/attribute/TransactionTypeAttributeDescriptionNGTest.java:22]: unused import

File: CoreAnalyticSchema/test/unit/src/au/gov/asd/tac/constellation/graph/schema/analytic/attribute/TransactionTypeAttributeDescriptionNGTest.java [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.graph.schema.analytic.attribute;
+
+import au.gov.asd.tac.constellation.graph.schema.analytic.concept.AnalyticConcept;
+import au.gov.asd.tac.constellation.graph.value.readables.IntReadable;
```
Comment (OrionsGuardian) [inline CoreAnalyticSchema/test/unit/src/au/gov/asd/tac/constellation/graph/schema/analytic/attribute/TransactionTypeAttributeDescriptionNGTest.java:19]: unused import

**Review by OrionsGuardian — CHANGES_REQUESTED** [review]

File: CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/attribute/objects/ContractedCompositeNodeState.java [hunk 5]
```diff
@@ -145,7 +145,7 @@ public List<Integer> expand(final GraphWriteMethods wg, final int vxId) {
             wg.setObjectValue(compositeStateAttr, id, new CompositeNodeState(id, expansionState));
         });
 
-        // Correct the x,y,z coordinates of the nodes in the consitutentNodeStore in case the composite node has been moved.
+        // Correct the x,y,z coordinates of the nodes in the constitutentNodeStore in case the composite node has been moved.
```
Comment (OrionsGuardian) [inline CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/attribute/objects/ContractedCompositeNodeState.java:148]: There's still a typo (an extra 't') in: `constituent`

**Review by OrionsGuardian — CHANGES_REQUESTED** [review]

**Review by OrionsGuardian — APPROVED** [review]

**Review by Delphinus8821 — APPROVED** [review]

**Pull request merged**